src/modules/LanItems.rb

```diff
@@ -1983,8 +1985,14 @@ def SetItem
 
       @description = BuildDescription(@type, @device, devmap, @Hardware)
 
-      SetDeviceVars(devmap, @SysconfigDefaults)
-      SetS390Vars(s390_devmap, @s390_defaults)
+      devmap = SetDeviceVars(devmap, @SysconfigDefaults)
+      s390_devmap = SetS390Vars(s390_devmap, @s390_defaults)
+
+      # FIXME: should raise an exception
```

Comment:
in fact not needed. Simply remove default param in API. So `def SetItem(builder:)` and it will raise it for you that param is missing.

Reply:
done